packages/workflow/src/AugmentObject.ts

```diff
@@ -102,11 +105,11 @@ export function augmentObject<T extends object>(data: T): T {
 
 			return value;
 		},
-		deleteProperty(target, key: string) {
-			if (key in newData) {
+		deleteProperty(_target, key: string) {
+			if (hasOwnProperty.call(newData, key)) {
 				delete newData[key];
 			}
-			if (key in target) {
+			if (hasOwnProperty.call(data, key)) {
```

Comment:
This now uses `data` instead of `target`. Is that change intentional? I assume they should refer to the same object, right?

Reply:
yeah. I found using `data` and `newData` less confusing than `target` and `newData`.